components/footer.js

```diff
@@ -12,8 +12,34 @@ import styles from "./style.module.css"
 export default function Footer() {
   return (
     <footer className={styles.footer}>
+    <div className={styles.bajo}>
```

Comment:
plase remove spanish names on variables

Reply:
i think what i changed all of variables, next project i will be careful with this error